modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java

```diff
@@ -341,9 +505,59 @@ private String _getProperty(
 	private Object _environment;
 	private final Gradle _gradle;
 	private Object _homeDir;
+	private final Object _liferayProductVersion;
+	private final ProductInfo _productInfo;
 	private final Set<ProjectConfigurator> _projectConfigurators =
 		new HashSet<>();
+	private final Object _releaseDate;
 	private final Plugin<Project> _rootProjectConfigurator;
 	private Object _targetPlatformVersion;
+	private Object _workspaceProductKey;
```

Comment:
```suggestion
	private Object _product;
```

This way it matches the naming scheme of everything else.